templates/todo/projects/python-mongo/.repo/terraform/infra/main.tf

```diff
@@ -0,0 +1,306 @@
+# Deploy app service plan
+# ------------------------------------------------------------------------------------------------------
+resource "azurecaf_name" "plan_name" {
+  name          = random_string.resource_token.result
+  resource_type = "azurerm_app_service_plan"
+  random_length = 0
+  clean_input   = true
+}
+resource "azurerm_service_plan" "plan" {
+  name                = azurecaf_name.plan_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  os_type             = "Linux"
+  sku_name            = "B1"
+
+  tags = local.tags
+}
+
+# ------------------------------------------------------------------------------------------------------
+# Deploy app service web app
+# ------------------------------------------------------------------------------------------------------
+resource "azurecaf_name" "web_name" {
+  name          = "web-${random_string.resource_token.result}"
+  resource_type = "azurerm_app_service"
+  random_length = 0
+  clean_input   = true
+}
+resource "azurerm_linux_web_app" "web" {
+  name                = azurecaf_name.web_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  service_plan_id     = azurerm_service_plan.plan.id
+  https_only          = true
+  tags                = merge(local.tags, { azd-service-name : "web" })
+
+  site_config {
+    #linux_fx_version = "NODE|16-lts"
+    always_on        = true
+    ftps_state       = "FtpsOnly"
+    app_command_line = "pm2 serve /home/site/wwwroot --no-daemon --spa"
+  }
+
+
+  app_settings = {
+    "SCM_DO_BUILD_DURING_DEPLOYMENT"        = "false"
+    "APPLICATIONINSIGHTS_CONNECTION_STRING" = module.applicationinsights.APPLICATIONINSIGHTS_CONNECTION_STRING
+  }
+
+  logs {
+    application_logs {
+      file_system_level = "Verbose"
+    }
+    detailed_error_messages = true
+    failed_request_tracing  = true
+    http_logs {
+      file_system {
+        retention_in_days = 1
+        retention_in_mb   = 35
+      }
+    }
+  }
+
+}
+
+# ------------------------------------------------------------------------------------------------------
+# Deploy app service api
+# ------------------------------------------------------------------------------------------------------
+resource "azurecaf_name" "appi_name" {
+  name          = "api-${random_string.resource_token.result}"
+  resource_type = "azurerm_app_service"
+  random_length = 0
+  clean_input   = true
+}
+
+
+resource "azurerm_linux_web_app" "api" {
+  name                = azurecaf_name.appi_name.result
+  location            = azurerm_resource_group.rg.location
+  resource_group_name = azurerm_resource_group.rg.name
+  service_plan_id     = azurerm_service_plan.plan.id
+  https_only          = true
+  tags                = merge(local.tags, { "azd-service-name" : "api" })
+
+  site_config {
+    #linux_fx_version = "PYTHON|3.8"

```

Comment:
Similar feedback as above, should we remove this in favor of setting `python_version` in the `application_stack` block?

Reply:
added the config